### PR TITLE
Fifth milestone

### PR DESCRIPTION
**Pull request by dannisonarias**
Turn App into Class Component
Implement the handleClick function
Tie logic to Components

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -1,44 +1,54 @@
...
+  };
+
   return (
     <div>
       <div style={styles}>
-        <Button color="#c9c5c5" name="AC" />
-        <Button color="#c9c5c5" name="+" />
-        <Button color="#c9c5c5" name="%" />
-        <Button name="÷" />
+        <Button color="#c9c5c5" name="AC" onClick={clickHandler} />
```
Comment (Oghenebrume50) [inline src/components/ButtonPanel.js:18]: [optional] Considering that 'onClick' is the name of react's default button event listener you might want to consider renaming for readability

File: src/logic/operate.js [hunk 2]
```diff
@@ -8,15 +8,14 @@ const operate = (numberOne, numberTwo, operation) => {
...
     case '+':
-      return (nOne + nTwo);
+      return (nOne.plus(nTwo)).toString();
     case 'X':
       return nOne.times(nTwo).toString();
-    case '÷':
+    case '/':
       if (nTwo === '0') {
-        return '0';
+        return Infinity;
```
Comment (Oghenebrume50) [inline src/logic/operate.js]: This doesn't handle the division by zero error, I still get an error when I try dividing by zero

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @dannisonarias you did a really good,  a good part of this program works fine, please see suggested changes below

There shoulg be a '+/-' button that flips the sign of a number, i.e if 45 is the displayed number on click it should be -45 and on another click it should go back to 45.

The percentage button is not working at the moment, it should return the percentage of any number currently being displayed at the time of clicking it, for context this works when you input something and just press the button, but using an already calculated value does not work

Please, work on these things and request for another review :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask:

**Review by shloch — APPROVED** [review]
## Hello @dannisonarias ,


calc

Good to go... THUMBS UP

**Pull request merged**